datasets/radarsat-1.yaml

```diff
@@ -0,0 +1,64 @@
+Name: RADARSAT-1 OPEN DATA COG PRODUCTS
+Description: Developed and operated by the Canadian Space Agency, it is Canada's first commercial Earth observation satellite.
+Documentation: https://mdacorporation.com/geospatial/international/satellites/RADARSAT-1/
+Contact: https://www.eodms-sgdot.nrcan-rncan.gc.ca
+ManagedBy: "[Natural Resources Canada](https://nrcan.gc.ca/)"
+UpdateFrequency: The initial product release includes the 36000 products originally ordered by the Government of Canada. Products are added on an adhoc basis driven by prioritized foreign repatriation efforts and new processing orders from the raw archive.
+Tags:
+  - earth observation
+  - global
+  - government spending
+  - ice
+  - agriculture
+  - sustainability
+  - disaster response
+  - satellite imagery
+  - geospatial
+License: [Open Government License (OGL)](https://open.canada.ca/en/open-government-licence-canada)
+Resources:
+  - Description: Cloud Optimized GeoTIFF scenes and metadata in a Requester Pays S3 bucket
+    ARN: arn:aws:s3:::radarsat-r1-l1-cog
+    Region: ca-central-1
+    Type: S3 Bucket
+    RequesterPays: True
+DataAtWork:
+  Tutorials:
+    - Title: PCI Geomatics Webinar | Cloud enabling earth observation archives
+      URL: https://www.youtube.com/watch?v=SvejDH5-Hic
+      AuthorName: PCI Geomatics
+      AuthorURL: https://www.pcigeomatics.com/
+      Services:
+  Tools & Applications:
+    - Title: RADARSTAC
+      URL: https://www.radarstac.com
+      AuthorName: Sparkgeo Consulting Inc. 
+      AuthorURL: https://sparkgeo.com/
+    - Title: QGIS
+      URL: https://qgis.org
+      AuthorName: German QGIS user group
+      AuthorURL: https://www.qgis.de/
+    - Title: GDAL
+      URL: https://gdal.org
+      AuthorName:

```

Comment:
For each of these, you will need at least an AuthorName (can be a company name).